components/barChart.js

```diff
@@ -276,16 +276,20 @@ export default function BarChartScreen({ drillData, drillInfo }) {
                   style={{ pointerEvents: "none" }}
                 />
               </BarChart>
-              <BarChart
-                style={styles.barChart}
-                data={transparentData}
-                svg={{ fill }}
-                contentInset={{
-                  left: halfScreenCompensation,
-                  right: halfScreenCompensation,
-                }}
-                yAccessor={({ item }) => item.value}
-              ></BarChart>
+              {/*<BarChart*/}
```

Comment:
Should this commented out code be removed if it won't be used?

Reply:
yes

Reply:
I'll do it in my next pr